packages/common/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@latticexyz/common",
+  "version": "1.42.0",
+  "description": "Common low level logic shared between packages",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/common"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./codegen": "./dist/codegen/index.js",
+    "./foundry": "./dist/foundry/index.js"
```

Comment:
👏  this is nice, so we can import just what we need

I wonder if we should have some policy around deps for this package, to keep it slim/focused